geode-core/build.gradle

```diff
@@ -301,3 +301,13 @@ distributedTest {
   // Some tests have inner tests that should be ignored
   exclude "**/*\$*.class"
 }
+
+afterEvaluate {
+  rootProject.generate.dependsOn(generateGrammarSource)
+  project.assemble.dependsOn(generateGrammarSource)
```

Comment:
Why do we need this dependency and all the others added after this line?

Reply:
`assemble` is the lifecycle task most closely aligned with this task as a dependency, then we use `mustRunAfter` to ensure that our generate tasks happen first within this project in all cases.

Reply:
I don’t understand why we need to do that though. They already happen before assemble by design. We should only need to add a pseudo target to encourage them to run outside of that normal behavior for quick project generation.

Reply:
@pivotal-jbarrett I'll remove the offending lines. Look for incoming commit...